static/js/task-feed/controllers/task-feed.controller.js

```diff
@@ -27,8 +27,44 @@
       self.modules = [];
 
       TaskFeed.getProjects().then(
-        function success (data) {
-            self.modules = data[0];
+        function success (successData) {
```

Comment:
the checks have to happen in backend, this is not the way to do it in front end, either way the the logic of checking here is wrong